### ISSUE TITLE
Grid does not have value for Name property

### ISSUE THREAD

**Issue by ylux587** [telerik/kendo-ui-core#5908]
### Bug report

**For Support request - Please do not submit support request here, instead see the** [Kendo UI Premium Forums](http://www.telerik.com/forums/kendo-ui) or our [support system](http://www.telerik.com/support) **at Telerik.com** !

**For Feature Requests - please submit those into [our feedback portal](https://www.telerik.com/support/feedback)**

### Reproduction of the problem
render a grid with some columns and data row. Use screen reader to read it

### Current behavior
For the  Table, Name property does not exist.
![image](https://user-images.githubusercontent.com/10621694/86951013-cefe7000-c105-11ea-9549-910b64fa1536.png)

### Expected/desired behavior
Name property should be provide for the table and screen reader should announces the same.

### Environment

* **Kendo UI version:** Kendo UI version: 2020.2.513
* **jQuery version:** 3.5.1
* **Browser:**Chromium Edge Version: 84.0.488.1 (Official build) dev (64-bit)

**Comment by GaloisGirl** [comment]
It's the programmer's job to set an `aria-label` or `aria-labelledby`, see https://www.w3.org/TR/wai-aria-practices-1.2/#grid .

**Comment by dmanova** [comment]
Consider the issue as answered with the previous comment and closing it.
